platforms/core-execution/build-cache-local/build.gradle.kts

```diff
@@ -6,13 +6,10 @@ plugins {
 description = "Local build cache implementation"
 
 dependencies {
-    api(project(":build-cache"))
-    api(libs.inject)
-
     api(project(":base-annotations"))
     api(project(":base-services"))
+    api(project(":build-cache"))
     api(project(":build-cache-spi"))
-    api(project(":core-api"))
```

Comment:
💭 This is important: our local build cache implementation doesn't depend on the whole public Gradle API anymore.